### DISCUSSION TITLE
Docker container for Arm architecture (aarch64 / arm64)

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2526] [Q&A] [answered]
Hello there. Is there a Contiki Docker container for Arm architecture (aarch64 / arm64)?

Thanks in advance. Rahav

**Comment by pjonsson** [comment, marked as answer]
No, no image for ARM unfortunately, Github does not yet provide CI runners with ARM.

If you're after ARM support in general I just made a [PR](https://github.com/contiki-ng/contiki-ng/pull/2535) for the CI to also build things with macOS, but it's still only x86-64. There's also another [PR](https://github.com/contiki-ng/contiki-ng/pull/2486) for M1/M2 support for Cooja.

**Reply by rahav** [reply]
Thank you
